apps/prairielearn/src/lib/workspace.ts

```diff
@@ -391,6 +391,20 @@ export async function initialize(workspace_id: string): Promise<InitializeResult
     targetPath: sourcePath,
   });
 
+  // Update permissions so that the directory and all contents are owned by the workspace user
+  await fsPromises.chown(
+    sourcePath,
+    config.workspaceJobsDirectoryOwnerUid,
+    config.workspaceJobsDirectoryOwnerGid,
+  );
```

Comment:
This is actually not needed because `klaw` includes the directory itself, so this is covered by the loop below.

Reply:
Good call, removed in 8b6212b.